src/components/UpcomingBirthdays/UpcomingBirthdays.js

```diff
@@ -1,9 +1,40 @@
 import './UpcomingBirthdays.css';
+import { useSelector } from 'react-redux';
 
 const UpcomingBirthdays = () => {
+  const friends = useSelector(state => state.friends);
+  const upcomingBirthdays = friends.filter(friend => {
+    const today = new Date();
+    const monthFromToday = new Date(today.getTime() + 30*24*60*60*1000);
+    const currentDate = today.toISOString().split('T')[0];
+    const currentMonthFromToday = monthFromToday.toISOString().split('T')[0];
+    return friend.birthday > currentDate && friend.birthday < currentMonthFromToday
+  });
+  const sortedBirthdays = upcomingBirthdays.sort((a, b) => new Date(a.birthday) - new Date(b.birthday))
+  const relevantBirthdays = sortedBirthdays.map(friend => {
+    return (
+      <article key={friend.id}>
+        <ul className='name-area'>
+          {friend.gift === 'purchased ' &&
+          <div className='check-area'>
+            <i className="fas fa-check">{friend.name}</i>
+          </div>}
+          {friend.gift === 'pending ' && <li>{friend.name}</li>}
+        </ul>
+          <ul>
+            <li>{friend.birthday.split('2021-')[1]}</li>
```

Comment:
I think I prefer displaying months with their names instead of the numbers, in the future we could potentially switch the bday displays to be like "July 7th" to be more friendly to read

Reply:
Yeah we could make this switch!